### PR TITLE
Updates on TMJets algorithms interface

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

File: src/Algorithms/TMJets/TMJets21b/TMJets21b.jl [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    absorb_steps::Int=3
+    disjointness::DM=ZonotopeEnclosure()
+end
+
+"""
+    TMJets
+
+The algorithm TMJets defaults to `TMJets21b`.
+"""
+const TMJets = TMJets21b
```
Comment (schillic) [inline src/Algorithms/TMJets/TMJets21b/TMJets21b.jl:26]: I would move this to a different file (mayb `common.jl`).

**Review by schillic — APPROVED** [review]

**Comment by mforets** [comment]
@lbenet @UzielLinares This PR brings the new TaylorModels.jl algorithms to the common ReachabilityAnalysis.jl interface:

- `TMJets21a()` calls `validated_integ` from TaylorModels.jl
- `TMJets21b()` calls `validated_integ2` from TaylorModels.jl
- `TMJets` is an alias for `TMJets21b`
- `TMJets20()` is the (old) adaptation of `validated_integ` implemented ReachabilityAnalysis.jl. I'm keeping it for the time being, for experimentation purposes.

**Comment by mforets** [comment]
We could also rename the new algorithm to `BL21` (for your last names) now or later. I was trying to make the tiniest update so that previous codes still work.

**Comment by schillic** [comment]
`BL21` would be consistent with the other algorithms.

**Pull request merged**